<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalInvoice</class>
 <widget class="QWidget" name="FinalInvoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>673</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="adminLoginLabel_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Full Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_18">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount Paid</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="totaltextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Method</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Payment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mainlabal_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 19pt &quot;Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Invoice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discoutpushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Booked Seats Number</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tickettextEdit_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TIckets</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
